build.gradle

```diff
@@ -5589,6 +5589,12 @@ compileTargets { t ->
         def standaloneSdkDir = "${rootProject.buildDir}/${standaloneSdkDirName}"
         def standaloneLegalDir = "${standaloneSdkDir}/legal"
 
+        def excludeDLLs = []
```

Comment:
maybe this should be renamed to excludeDynamicLibs or so, as I think (fear) this issue might surface for other platforms in the future as well.

Reply:
OK, I change it to excludeNativeLibs. The code that populates the list is still specific to Microsoft DLLs, but could be generalized later if there is a need.